internal/services/sentinel/sentinel_watchlist_resource.go

```diff
@@ -131,10 +131,6 @@ func (r WatchlistResource) Create() sdk.ResourceFunc {
 					// The only supported provider for now is "Microsoft"
 					Provider: utils.String("Microsoft"),
 
-					// The "source" represent the source file name which contains the watchlist items.
```

Comment:
Is the `Source` removed from the new API now?

Reply:
There does still be a `Source` property, while it seems the code about Source here only "makes API happy" without other targets. `acctests` passed without these lines, so I think it's better to remove them.

Reply:
Sounds reasonable.